.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7, 3.8 ]
+        python-version: [ 3.6, 3.7, 3.8, 3.9 ]
```

Comment:
can you remove 3.6?

Reply:
in another PR. dropping 3.6 here and also modifying setup.py